components/pkg-export-helm/src/main.rs

```diff
@@ -130,6 +130,15 @@ fn cli<'a, 'b>() -> clap::App<'a, 'b> {
                 .validator(valid_url)
                 .help("A URL to an SVG or PNG image to be used as an icon"),
         )
+        .arg(
+            Arg::with_name("SOURCE")
+                .value_name("URL")
+                .long("source")
+                .short("s")
+                .multiple(true)
+                .validator(valid_url)
+                .help("URL to source code for the project"),
```

Comment:
Elsewhere you use "A URL", so you might wanna be consistent.

Also I'd say "to *the*".